src/System.Globalization.Extensions/tests/IdnMapping/IdnMappingGetAsciiTests.cs

```diff
@@ -83,24 +83,29 @@ public void GetAscii(string unicode, int index, int count, string expected)
             }
             Assert.Equal(expected, new IdnMapping().GetAscii(unicode, index, count));
         }
-        
-        [Fact]
-        public void TestGetAsciiWithDot()
+
+        public static bool GetAsciiWithDotSucceeds => !GetAsciiWithDotFails;
+
+        [ConditionalFact(nameof(GetAsciiWithDotSucceeds))]
+        public void TestGetAsciiWithDot_Succeeds()
         {
-            string result = "";
-            Exception ex = Record.Exception(()=> result = new IdnMapping().GetAscii("."));
-            
-            if (ex == null)
-            {
-                // Windows and OSX always throw exception. some versions of Linux succeed and others throw exception   
-                Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Windows));
-                Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.OSX));
-                Assert.Equal(result, ".");
-            }
-            else
-            {
-                Assert.True(ex is ArgumentException);
-            }
+            string dot = ".";
+            string result = new IdnMapping().GetAscii(dot);
+            Assert.Equal(dot, result);
+        }
+
+        public static bool GetAsciiWithDotFails =>
+            PlatformDetection.IsWindows ||
+            PlatformDetection.IsOSX ||
+            PlatformDetection.IsUbuntu1604 ||
+            PlatformDetection.IsUbuntu1610 ||
+            RuntimeInformation.IsOSPlatform(OSPlatform.Create("openSUSE"));
```

Comment:
>RuntimeInformation.IsOSPlatform [](start = 12, length = 31)

nit: could you please add a property in PlatfornDetection (under the common folder) called IsOpenSUSE and then use it here?